project/Dependencies.scala

```diff
@@ -0,0 +1,24 @@
+import sbt._
+
+object Dependencies {
```

Comment:
I'm not sure you gain much by splitting dependencies out of build.sbt at this point? The project (and it's configuration) is small enough to reside in a single `build.sbt` file.

Reply:
The sbt documentation (to which I'm new) said that it's a best practice to split dependencies into their own class. I don't mind merging it into `Build.scala` or `build.sbt`, although it did feel a bit cleaner to separate them.